ash/src/lib.rs

```diff
@@ -59,6 +65,8 @@ pub mod vk;
 /// Wrappers for Vulkan extensions
 pub mod extensions;
 
+pub(crate) mod prelude_internal;
```

Comment:
really could do with a better name

Reply:
Yeah, I was thinking the same, not sure what to call it though. Perhaps folding this in with `prelude` and only exporting these items as `pub(crate)` might be enough?

After all 60% of `ash/src/prelude.rs` (_not_ `ash/src/vk/prelude.rs`) is already `pub(crate)` for internal use only.

But if you do, a submodule of that prelude might be nicest, since it's a wildcard reexport.